spec/models/contact_spec.rb

```diff
@@ -2,136 +2,75 @@
-            before { subject.phone = '' }
-
-            it { expect(subject.valid?).to be false }
-          end
-        end
-      end
     end
 
-    describe 'post_option' do
+    describe 'feedback_opt_in' do
       context 'when true' do
         it { expect(subject.valid?).to be true }
       end
 
       context 'when false' do
-        before { subject.post_option = false }
+        before { subject.feedback_opt_in = false }
 
         it { expect(subject.valid?).to be true }
       end
     end
-
-    describe 'when no option is picked' do
-      # TODO: discuss this
-      before do
-        subject.email_option = false
-        subject.phone_option = false
-        subject.post_option = false
-      end
-    end
   end
 
   describe '#export' do
-    let(:email_option) { true }
     let(:email) { 'some@email.domain' }
-    let(:phone_option) { true }
-    let(:phone) { '12345678' }
-    let(:post_option) { true }
+    let(:feedback_opt_in) { true }
 
-    subject { described_class.new(email_option: email_option, email: email, phone_option: phone_option, phone: phone, post_option: post_option).export }
+    subject { described_class.new(email: email, feedback_opt_in: feedback_opt_in).export }
 
-    context 'when email_option is true' do
+    context 'when email is set' do
       it 'the returned hash includes email_contact true and email_address' do
         is_expected.to include(email_contact: true, email_address: email)
       end
     end
 
-    context 'when email_option is false' do
-      let(:email_option) { false }
+    context 'when email is not set' do
+      let(:email) { nil }
 
-      it 'the returned hash includes email_contact true and email_address is nil' do
-        is_expected.to include(email_contact: false, email_address: nil)
+      it 'the returned hash includes email_contact false and email_address is nil' do
+        is_expected.to include(email_contact: false, email_address: email)
       end
     end
 
-    context 'when phone_option is true' do
-      it 'the returned hash includes phone_contact true and phone' do
-        is_expected.to include(phone_contact: true, phone: phone)
-      end
-    end
-
-    context 'when phone_option is false' do
-      let(:phone_option) { false }
-
-      it 'the returned hash includes phone_contact true and phone is nil' do
-        is_expected.to include(phone_contact: false, phone: nil)
-      end
+    it 'the returned hash always returns phone_contact as false' do
+      is_expected.to include(phone_contact: false)
     end
 
-    context 'when post_option is true' do
-      it 'the returned hash includes post_contact true' do
-        is_expected.to include(post_contact: true)
-      end
+    it 'the returned hash includesreturns post_contact as false' do
+      is_expected.to include(post_contact: false)
     end
 
-    context 'when post_option is false' do
-      let(:post_option) { false }
+    context 'when feedback_opt_in is false' do
```

Comment:
We should test for when `feedback_opt_in` is `true` as well, to make sure it's exported well.